packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/disabilityAssessments.js

```diff
@@ -20,10 +20,37 @@ const potentiallyHarmfulTwoAlternatives = "Avoid using <i>%1$s</i> as it is pote
 const disabilityAssessments =  [
 	{
 		identifier: "binge",
-		nonInclusivePhrases: [ "bingeing", "binge" ],
-		inclusiveAlternatives: "<i>indulging, satuating, wallowing, spree, marathon</i>",
+		nonInclusivePhrases: [ "binge" ],
+		inclusiveAlternatives: "<i>indulge, satuate, wallow, spree, marathon, consume excessively</i>",
 		score: SCORES.POTENTIALLY_NON_INCLUSIVE,
-		feedbackFormat: medicalCondition,
+		feedbackFormat: "Avoid using <i>%1$s</i>, unless talking about a symptom of a medical condition. " +
+			"If you are not referencing a medical condition, consider other alternatives to describe the trait or behavior, such as %2$s.",
+		rule: ( words, nonInclusivePhrase ) => includesConsecutiveWords( words, nonInclusivePhrase )
+			.filter( isNotFollowedByException( words, nonInclusivePhrase, [ "drink", "drinks", "drinking" ] ) ),
+	},
+	{
+		identifier: "bingeing",
+		nonInclusivePhrases: [ "bingeing" ],
```

Comment:
We should also target _binging_, as [both spellings are acceptable](https://www.grammarly.com/blog/bingeing-binging/).